### PR TITLE
Update Test/Release Group Meeting Time

### PR DESCRIPTION
**Pull request by hklai**

### CODE REVIEW

File: WORKING-GROUPS.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ Build, test, release
 Artifact | Link
 ---|---
 Forum | [istio-test-release@](https://groups.google.com/forum/#!forum/istio-test-release)
-Community Meeting VC | [Thursdays at 13:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
+Community Meeting VC | [Monday at 10:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
```
Comment (ldemailly) [inline WORKING-GROUPS.md]: isn't it at 11a pst ? (10 is the biweekly leads one?)

File: WORKING-GROUPS.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ Build, test, release
...
 Community Meeting Calendar | [Calendar Invitation](https://calendar.google.com/event?action=TEMPLATE&tmeid=YjkwZGxpaGFpajJnMDZpb2w0a3Qxa2kwdmdfMjAxNzExMzBUMjEwMDAwWiBoa2xhaUBnb29nbGUuY29t&tmsrc=hklai%40google.com)
```
Comment (ldemailly) [inline WORKING-GROUPS.md]: this still points to Thu 1pm

Comment (hklai) [reply WORKING-GROUPS.md]: Thanks. The old one was actually a past instance, and was not even recurring. =)

File: WORKING-GROUPS.md (hunk 1 above)
Comment (hklai) [reply WORKING-GROUPS.md]: Thanks

**Comment by hklai** [comment]
Comments addressed. PTAL

**Pull request merged**